examples/all/proximity_sensor_example.cc

```diff
@@ -92,17 +92,17 @@ static uint8_t read_proximity_sensor(Mmio<OpenTitanI2c> i2c)
 		i2c->speed_set(1);
 	};
 
-	auto i2c0 = MMIO_CAPABILITY(OpenTitanI2c, i2c0);
-	i2cSetup(i2c0);
+	auto i2c1 = MMIO_CAPABILITY(OpenTitanI2c, i2c1);
+	i2cSetup(i2c1);
 	uint8_t addr;
 
 	auto rgbled = MMIO_CAPABILITY(SonataRgbLedController, rgbled);
 
-	setup_proximity_sensor(i2c0, ApdS9960I2cAddress);
+	setup_proximity_sensor(i2c1, ApdS9960I2cAddress);
 
 	while (true)
 	{
-		uint8_t prox = read_proximity_sensor(i2c0);
+		uint8_t prox = read_proximity_sensor(i2c1);
```

Comment:
Is changing the device required, can't one just plug the device into the other qwiic socket that's connected to ic0?